src/util/metaData.ts

```diff
@@ -0,0 +1,39 @@
+import AjvModule from 'ajv'
+
+const Ajv = AjvModule.default
+const ajv = new Ajv({ allErrors: true })
+
+const schema = {
+  type: 'object',
+  properties: {
+    Creator: { type: 'string', minLength: 1 },
+    Description: { type: 'string', maxLength: 80, minLength: 1 },
+    Language: { type: 'string', minLength: 1 },
+    Publisher: { type: 'string', minLength: 1 },
+    Title: { type: 'string', maxLength: 30, minLength: 1 },
+  },
+  required: ['Creator', 'Description', 'Language', 'Publisher', 'Title'],
```

Comment:
Should be handled in properties as a `boolean`

Reply:
In this case, I used the Ajv JSON schema validator. For this library, the required fields should be passed as an array.

Reply:
OK... Not sure this is the right tooling... but let deal with it.